scripts/kind/tools/ingress.sh

```diff
@@ -2,19 +2,19 @@
 
 op=$1
 
-source ${ROOT_DIR}/hack/common.sh
+source ${REPOROOT}/scripts/kind/common.sh
```

Comment:
can you define a local variable `{REPOROOT}/scripts/kind`, we have too many references to it.

Reply:
fixed. Thanks